kids-learning-app/assets/js/app.js

```diff
@@ -0,0 +1,260 @@
+const components = {
+    bodyContainer : () => {
+        let body = document.querySelectorAll(".body-flex")[0]; ///changes....var => let
+        body.innerHTML = "";
+        return body;
+    },
+
+    titleDiv : () => {
+        let title = document.createElement('div');
+        title.setAttribute("class","title");
+        return title;
+    },
+
+    checkIcon: (str) => {
+        let chkicon = document.createElement('i');
+        chkicon.classList.add("fa-solid");
+        chkicon.classList.add("fa-check");
+        chkicon.classList.add(str);
+        chkicon.onclick = (event) => {
+            event.preventDefault();
+        };
+        return chkicon;
+    },
+
+    circleButton : () => {
+        let circle = document.createElement('div');
+        circle.setAttribute("class","circle");
+        return circle;
+    },
+
+    triangleButton : () => {
+        let triangle = document.createElement('div');
+        triangle.setAttribute("class","triangle");
+        return triangle;
+    },
+
+    squareButton : () => {
+        let square = document.createElement('div');
+        square.setAttribute("class","square");
+        return square;
+    },
```

Comment:
this approach again requires code change which can be made generic for minimal code change